content/java/concepts/stringbuilder/terms/insert/insert.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.insert()'
+Description: 'Inserts a sequence of characters into the StringBuilder.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Characters'
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.insert()`** method inserts a sequence of characters into the `StringBuilder`. It returns a reference to the `StringBuilder` object.
+
+## Syntax
+
+```pseudo
+myStringBuilder.insert(index, str);
+```
+
+The `index` argument is the zero-based index where the characters are to be inserted. If `str` is a [`String`](https://www.codecademy.com/resources/docs/java/strings), a `CharSequence`, or a `char[]` array, the characters contained are inserted at `index` and its capacity is increased by the number of characters inserted. For other types, it behaves as if `str` was first converted to a string by using `String.valueOf(str)`. If `index` is negative or greater than the length of the sequence then a [`StringIndexOutOfBoundsException`](https://www.codecademy.com/resources/docs/java/errors/stringindexoutofboundsexception) is thrown.
```

Comment:
```suggestion
The `index` argument is the zero-based index where the characters are to be inserted. If `str` is a [`String`](https://www.codecademy.com/resources/docs/java/strings), a `CharSequence`, or a `char[]` [array](https://www.codecademy.com/resources/docs/java/arrays), the characters contained are inserted at `index` and its capacity is increased by the number of characters inserted. For other types, it behaves as if `str` was first converted to a string by using `String.valueOf(str)`. If `index` is negative or greater than the length of the sequence, then a [`StringIndexOutOfBoundsException`](https://www.codecademy.com/resources/docs/java/errors/stringindexoutofboundsexception) is thrown.
```

Also wondering if it would make sense to add a single asterisk after "CharSequence" and a double-asterisk after "char[] array" (and then prepend with asterisks on lines # 25 and # 33, respectively)?